apps/web/src/server/views/app/layouts/app-two-column.layout.njk

```diff
@@ -21,11 +21,11 @@
 				}}
 			{% endif %}
 			{% block pageHeading %}
-					<h1 class="govuk-heading-xl">
-            {% if pageCaption %}
-              <span class="govuk-caption-xl">{{ pageCaption }}</span>
-            {% endif %}{{ pageHeading or pageTitle }}
-          </h1>{% endblock %}
+				<h1 class="govuk-heading-xl">
```

Comment:
This is just an indentation fix (I noticed the indentation was off and mixed tabs/spaces were being used) - no actual changes here